blog/next.config.js

```diff
@@ -3,11 +3,21 @@ const withCustomBabelConfig = require("next-plugin-custom-babel-config");
 const withTypescript = require("@zeit/next-typescript");
 const withTranspileModules = require("next-plugin-transpile-modules");
 
-module.exports = withCustomBabelConfig(
+// https://github.com/wellcometrust/next-plugin-transpile-modules/issues/11
+function fixExternalsPlugin(config = {}) {
```

Comment:
Hmm, seems as though this issue should be addressed up-stream in `next-plugin-transpile-modules`, but I'm happy to merge this in for now

Reply:
Added comments that it is workaround and it needs to be removed when module is fixed.